tests/foreman/ui/test_leapp_client.py

```diff
@@ -0,0 +1,79 @@
+"""Tests for leapp upgrade of content hosts with Satellite
+
+:Requirement: leapp
+
+:CaseComponent: Leappintegration
+
+:Team: Rocket
+
+:CaseImportance: Critical
+
+:CaseAutomation: Automated
+
+"""
+
+import pytest
+
+synced_repos = pytest.StashKey[dict]
```

Comment:
Doesnt seems its used anywhere in the module.

Reply:
I will remove this.